UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentDataService.cs

```diff
@@ -76,6 +79,7 @@ public async Task QueryAndDownloadFileShareServiceFiles(SalesCatalogueServiceRes
         }
         private async Task CreateAncillaryFiles(string batchId, string exchangeSetRootPath, string correlationId)
         {
+            await CreateSerialEncFile(batchId, exchangeSetRootPath, correlationId);
```

Comment:
Change parameter name passed from exchangeSetRootPath to exchangeSetPath.

Reply:
Resolved